lib/tbot/config/config.go

```diff
@@ -268,7 +269,8 @@ type BotConfig struct {
 	Services   Services         `yaml:"services,omitempty"`
 
 	Debug           bool          `yaml:"debug"`
-	AuthServer      string        `yaml:"auth_server"`
+	AuthServer      string        `yaml:"auth_server,omitempty"`
+	Proxy           string        `yaml:"proxy,omitempty"`
```

Comment:
```suggestion
	Proxy           string        `yaml:"proxy_server,omitempty"`
```

For consistency with teleport.yaml?